tests/apps/core/test_core_api_users.py

```diff
@@ -110,6 +111,7 @@ def test_api_users_list_authenticated_by_username(self):
                 "id": str(user1.id),
                 "language": user1.language,
                 "name": user1.name,
+                "timezone": user1.timezone.key,
```

Comment:
I think you should open an issue to remove the usage of variable in the assertion and replace them with their values like we do in other projects. WDYT ?

Reply:
Not sure why. I see value in random testing here no?